handsontable/src/shortcuts/recorder.js

```diff
@@ -124,6 +133,10 @@ export function useRecorder(ownerWindow, beforeKeyDown, afterKeyDown, callback)
    * @private
    */
   const onblur = () => {
+    if (handleEvent() === false) {
+      return;
+    }
+
```

Comment:
This line causes a bug where the Control/Command key is not released when the browser window is blurred. Thus, after the window is focused again the non-contiguous selection is locked even when the meta key is not pressed at all. 

![Kapture 2022-06-09 at 08 24 19](https://user-images.githubusercontent.com/571316/172779251-e7c1769d-6e91-4a75-a2e5-05585e14a96e.gif)